services/influxdb/service.go

```diff
@@ -563,6 +563,11 @@ func (c *influxdbCluster) Open() error {
 
 	c.watchSubs()
 
+	// Even if we're not linking subscriptions, validate the client to ensure influxdb is actually up
+	if err := c.validateClientWithBackoff(ctx); err != nil {
```

Comment:
Note this pings the configured influxdb and happens before we try to set up the flux task service.